common/constants.go

```diff
@@ -36,10 +36,18 @@ var TIPV2SwitchBlock = big.NewInt(0)
 
 var TIPNoHalvingMNReward = big.NewInt(0) // hardfork no halving masternodes reward
 var BlackListHFNumber = uint64(0)
-var TIPXDCX = big.NewInt(0)
-var TIPXDCXLending = big.NewInt(0)
-var TIPXDCXCancellationFee = big.NewInt(0)
-var TIPXDCXCancellationFeeTestnet = big.NewInt(0)
+var TIPXDCXDISABLE = big.NewInt(0)
+var TIPXDCX = big.NewInt(38383838)
+var TIPXDCXLending = big.NewInt(38383838)
+var TIPXDCXCancellationFee = big.NewInt(38383838)
+var TIPXDCXCancellationFeeTestnet = big.NewInt(38383838)
+var TIPXDCXMinerDisable = big.NewInt(88999999900)
```

Comment:
subnet shouldn't need all these constant on xdcx, no?

Reply:
It doesn't but I think in general the subnet code should be as close to xdpos as possible.
We could discuss more on this. Also, planning to do a diff documentation between xdpos and subnet.